src/js/components/stock-movement-wizard/AddItemsPage.jsx

```diff
@@ -419,6 +447,23 @@ class AddItemsPage extends Component {
         )
       ) {
         lineItemsToBeUpdated.push(item);
+      } else if (duplicatedItem) {
+        const duplicatedItemIndex = _.findIndex(
+          lineItemsToBeAdded,
```

Comment:
Why you look int `lineItemsToBeAdded` here?

Reply:
If there is item to add that has the same product that item to update it should not be added but it's quantity should be consolidated to item to update.